app/experimenter/nimbus-ui/src/AppLayout/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+
+type AppLayoutProps = {
+  children: React.ReactNode;
+};
+
+// TODO: Depending on the main page ("directory view") design, we may want two
+// AppLayouts - one with a header/footer, and another with a nav sidebar.
+
+export const AppLayout = ({ children }: AppLayoutProps) => {
+  return (
+    <main id="main" data-testid="main">
```

Comment:
Unless you're thinking we might add a Skip button I don't think we need this ID. #downwithids

Reply:
Omg especially if there might be multiple `AppLayout`s can you IMAGINE the scandal here??